features/issue.feature

```diff
@@ -80,6 +80,34 @@ Feature: hub issue
     """
     When I successfully run `hub issue -d 2016-08-18T09:11:32Z`
 
+  Scenario: Fetch issues
+    Given the GitHub API server:
+    """
+    get('/repos/github/hub/issues') {
+      assert :sort => "comments"
+      assert :direction => "asc"
+
+      json [
```

Comment:
OK, done. (my initial thought was that a naive implementation could have tried to sort the results locally, and I wanted to test against that)